python/ray/runtime_context.py

```diff
@@ -148,6 +240,15 @@ def current_placement_group_id(self):
         """
         return self.worker.placement_group_id
 
+    def get_current_placement_group_id(self) -> Optional[str]:
```

Comment:
For the consistency with other apis, should we rename it to `get_placement_group_id`?

Reply:
This is trying to match the existing current_placement_group_id() naming. Is that not as desirable?

Reply:
Since we are designing a new set of apis, it's better to be consistent among them. Thoughts @ericl ?

Reply:
No opinion here, either seems fine.